server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -394,6 +397,8 @@
     private VMSnapshotDao vmSnapshotDao;
     @Inject
     private BackupOfferingDao backupOfferingDao;
+    @Inject
+    private UserStatisticsDao _userStatsDao;
```

Comment:
@ravening can you please remove the underscore `_` at the beginning of the variable name?
Many variables on the codebase have `_` at the beginning but it is not aligned with the [Java variable naming convention](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html).

According to [Java variable naming convention](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html):

> A similar convention exists for the underscore character; while it's technically legal to begin your variable's name with "_", this practice is discouraged. White space is not permitted.

Reply:
@GabrielBrascher changed the code. please review again